### ISSUE TITLE
Direct AJAX links

### ISSUE THREAD

**Issue by lgatto** [lgatto/rols#7]
AJAX is a technology that allows complex interactions to a back-end server that are transparent to the user. In the case of the ontology browser, for example, selecting a term will update the metadata section of the display without reloading the whole page.
Application developers can take advantage of this without needing to install the OLS locally. The XML that is processed by the AJAX libraries can be accessed directly using specifically-formatted URLs:
- To obtain term metadata: `http://www.ebi.ac.uk/ontology-lookup/ajax.view?q=termmetadata&termid=[ID]&ontologyname=[LABEL]`
Where ID is a required term ID (eg: GO:0007067) and LABEL is the optional short label of an ontology (eg: GO).
- To obtain term name suggestions: `http://www.ebi.ac.uk/ontology-lookup/ajax.view?q=termautocomplete&termname=[NAME]&ontologyname=[LABEL]`
Where NAME is a required term name substring (eg: mito) and LABEL is the optional short label of an ontology (eg: GO). If left blank, the term names will be prefixed with the ontology label where they can be found.
- To obtain a full term name: `http://www.ebi.ac.uk/ontology-lookup/ajax.view?q=termname&termid=[ID]&ontologyname=[LABEL]`
Where ID is a required term ID (eg: GO:0007067) and LABEL is the optional short label of an ontology (eg: GO).
